package.json

```diff
@@ -50,5 +50,8 @@
   "bugs": {
     "url": "https://github.com/amasad/sane/issues"
   },
-  "homepage": "https://github.com/amasad/sane"
+  "homepage": "https://github.com/amasad/sane",
+  "optionalDependencies": {
+    "fsevents": "^1.1.1"
```

Comment:
I'm nervous to introduce the node-gyp dependency here. Although it is an optional dependency the ergonomics around failed optionalDependencies is still quite poor for those with funky environments. 

On a positive note, it looks like fsevents does seems to do its best to avoid pain here:

* only run for OSX https://github.com/strongloop/fsevents/blob/master/binding.gyp#L6
* only run in OSX https://github.com/strongloop/fsevents/blob/master/install.js#L1
*  uses `node-pre-gyp` to  install a binary when possible.

One last concern is if the binaries themselves are not stored in the registry, so companies with private registries (for insulation from the outside world) might become unhappy. For this reason, if we proceed I would like to recommend this be at-least a major version bump.